.github/workflows/github-labels.yml

```
@@ -0,0 +1,169 @@
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+      
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: 1.11.4  # Latest stable version
+      
+      - name: Configure Terraform
+        run: |
+          cd terraform/github/examples/labels
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)